### DISCUSSION TITLE
How does the latitude and longitude of EMANE correspond to XYZ of CORE?

### DISCUSSION THREAD

**Discussion by 1-Jone** [coreemu/core#724] [Q&A] [answered]
Hello. How does the latitude and longitude of EMANE correspond to XYZ of CORE?

**Comment by bharnden** [comment, marked as answer]
Depends, it doesn't have to translate at all. There are 2 configuration settings in `/etc/core/core.conf`

```
emane_event_generate = True
emane_event_monitor = False
```

One generates events that does an XY translation to lon,lat. One is for monitoring EMANE lon,lat to translate to XY. I would personally keep the XY as XY and leave GEO to EMANE.

But for that translation, that is done by some of the settings in the GUI. `Menu->Canvas->Size and Scale`. There you define how big the canvas is, how pixels translate to meters and the default reference point to translate. It is not a perfect system and CORE itself is not a great tool to actually render GEO by any means, but this is what has existed.
